website/content/docs/connect/gateways/api-gateway/configuration/http-route.mdx

```diff
@@ -533,6 +533,10 @@ Specifies the HTTP method to match.
 
 Specifies type of match for the path: `"exact"`, `"prefix"`, or `"regex"`.
 
+-> **Note:** Prefix matching uses simple string matching when comparing to the path. This differs slightly from the
+[Kubernetes Gateway API specification](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.PathMatchType)
+which declares that path prefix matching is done on a path element by element basis.
+
```

Comment:
```suggestion
-> **Note:** Prefix matching uses simple string matching when comparing to the path. This differs slightly from the
[Kubernetes Gateway API specification](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.PathMatchType)
which declares that path prefix matching is done on a path element by element basis. As
an example, configuring Consul with the prefix match string `/abc`, will result in the URL 
paths `/abc`, `/abc/`, `/abc/def`, and also `/abcd` being matched.

```
I think a lot of people will understand this faster by including an example.

Reply:
@Jeff-Apple does the change that Adam cover your concerns? It seems you both were wanting to add an example 💯

Reply:
Yes. And not surprisingly, he said it much better than I did.